CHANGELOG.md

```diff
@@ -659,7 +659,7 @@
     Walk symlinks when checking cached images for copied/added files
     ReserveSELinuxLabels(): handle wrapped errors from OpenBuilder
     Set WorkingDir to empty, not / for conformance
-    Update calls in e2e to addres 1101
+    Update calls in e2e to address 1101
```

Comment:
I'd prefer not changing this file, changelog.txt or buildah.spec for these.  Yep, it corrects the mis-spell, but it would be hard to find these commits if you went searching for them as the commit message is still misspelled.